core-bundle/tests/Monolog/ContaoTableProcessorTest.php

```diff
@@ -23,24 +23,48 @@
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
 
+/**
+ * @phpstan-import-type Record from Logger
+ */
 class ContaoTableProcessorTest extends TestCase
 {
     public function testCanBeInvoked(): void
     {
         $processor = $this->getContaoTableProcessor();
 
-        $this->assertEmpty($processor([]));
-        $this->assertSame(['context' => ['contao' => false]], $processor(['context' => ['contao' => false]]));
+        /** @phpstan-var Record $record */
+        $record = [];
+
+        $this->assertEmpty($processor($record));
+
+        $record = [
+            'level' => Logger::DEBUG,
+            'level_name' => 'DEBUG',
+            'channel' => '',
+            'context' => ['contao' => false],
+            'extra' => [],
+            'datetime' => new \DateTimeImmutable(),
+            'message' => '',
+        ];
+        $this->assertSame($record, $record);
```

Comment:
```suggestion

        $this->assertSame($record, $record);
```

Reply:
Done in fd17c9bc31874974972f48b4cad836553628d32d